README.md

```diff
@@ -105,175 +106,170 @@ There are [community driven install methods](./doc/unofficialInstallMethods.md)
+ - Hardened activation flow for x-cmdpal:// links with synchronous handling + timeout. Thanks [@jiripolasek](https://github.com/jiripolasek)!
+ - Enabled AOT by default for performance while simplifying publish configs.
+ - Cleaned and codified .editorconfig without behavior change. Thanks [@jiripolasek](https://github.com/jiripolasek)!
+ - Replaced service state color dots with play/pause/stop icons for accessibility. Thanks [@samrueby](https://github.com/samrueby)!
+ - Fixed filter dropdown sync and crash by binding SelectedValue and raising UI-thread notifications. Thanks [@jiripolasek](https://github.com/jiripolasek)!
+ - Ensured long links wrap correctly in details view.
+ - Removed animation + enforced min width on filter dropdown for clarity. Thanks [@jiripolasek](https://github.com/jiripolasek)!
+ - Restored focus to More button after ESC closes context menu improving keyboard flow.
+ - Marked main and toast windows as tool windows to keep them out of Alt+Tab while preserving style. Thanks [@jiripolasek](https://github.com/jiripolasek)!
+ - Fixed AOT template + theming issues for filter separators. Thanks [@jiripolasek](https://github.com/jiripolasek)!
+ - Introduced grid layouts (small/medium/gallery) for richer page presentation.
+ - Materialized result lists to avoid rescoring overhead.
+ - Disabled problematic selection TextToSuggest behind env flag.
+ - Resolved AOT warning by making IconMarginConverter partial. Thanks [@samrueby](https://github.com/samrueby)!
+ - Major search performance improvements (new fuzzy matcher, smarter fallbacks, fewer exceptions).
+ - Added context menu "Show Details" command when details pane hidden.
+ - Reduced window flicker by avoiding unnecessary cloaking. Thanks [@jiripolasek](https://github.com/jiripolasek)!
+ - Restored EmptyContent rendering for blank states. Thanks [@DevLGuilherme](https://github.com/DevLGuilherme)!
+ - Saved new state even if prior app state file corrupt (better resilience). Thanks [@jiripolasek](https://github.com/jiripolasek)!
+ - Migrated settings window to WinUI TitleBar control. Thanks [@jiripolasek](https://github.com/jiripolasek)!
+ - Prevented crash on duplicate keybindings and simplified matching. Thanks [@jiripolasek](https://github.com/jiripolasek)!
+ - Centralized fullscreen ignore-hotkey logic for summon handling. Thanks [@jiripolasek](https://github.com/jiripolasek)!
+ - Hid search box on content pages; improved focus and a11y; added Home title. Thanks [@jiripolasek](https://github.com/jiripolasek)!
+ - Synced generated resource comment to eliminate noisy diffs. Thanks [@jiripolasek](https://github.com/jiripolasek)!
+ - Blocked Ctrl+I from inserting stray tab in search box.
+ - Logged HRESULT codes in error logs for deeper diagnostics. Thanks [@jiripolasek](https://github.com/jiripolasek)!
+ - Advanced font/emoji icon classification & alignment improvements. Thanks [@jiripolasek](https://github.com/jiripolasek)!
```

Comment:
(Unsure whether this should be "...classification and alignment improvements" or "...classification, plus alignment improvements". Are the alignment improvements a separate matter?)

```suggestion
 - Advanced font / emoji icon classification and alignment improvements. Thanks [@jiripolasek](https://github.com/jiripolasek)!
```